pkg/transparentproxy/kubernetes/kubernetes_config.go

```diff
@@ -484,13 +484,11 @@ func ConfigToAnnotations(
 	cfg tproxy_config.Config,
 	runtimeCfg k8s.Injector,
 	annotations map[string]string,
-	mesh string,
 	defaultAdminPort uint32,
 ) (map[string]string, error) {
 	result := map[string]string{
 		k8s_metadata.KumaSidecarInjectedAnnotation:                 k8s_metadata.AnnotationTrue,
 		k8s_metadata.KumaTransparentProxyingAnnotation:             k8s_metadata.AnnotationEnabled,
-		k8s_metadata.KumaMeshAnnotation:                            mesh, // either user-defined value or default
```

Comment:
does this need to be now be put in labels by transparentproxy? :thinking:

Reply:
Yes, It was called by `injector.go` and now we add it there

Reply:
OK got it